packages/web-components/fast-html/src/fixtures/dot-syntax/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+import path from "path";
+import { fileURLToPath } from "url";
+import { merge } from "webpack-merge";
+import config from "../../../webpack.common.config.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.resolve(path.dirname(__filename), "./");
+
+export default merge(config, {
```

Comment:
Ahh, we need to consider using vite/esbuild/swc, webpack adds so much bloat to the output.

Reply:
It's only for testing, we don't use it for any kind of output, we will use esbuild/rollup for a minified file once we get to a state we like, at which point I may switch to vite for the testing (webpack is what we are using everywhere else and I would prefer to change it all at once to prevent dev dependency bloat).